service/kvmconfig/v4/resource/deployment/master_deployment.go

```diff
@@ -186,6 +186,10 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 										Name:  "CORES",
 										Value: fmt.Sprintf("%d", capabilities.CPUs),
 									},
+									{
+										Name:  "COREOS_VERSION",
+										Value: key.CoreosVersion,
```

Comment:
Where is this var used?

Reply:
https://github.com/giantswarm/k8s-kvm/blob/master/docker-entrypoint.sh#L71-L103

Reply:
Ack. Thanks.